javascript/javascript2/week1/homework/cardio.js

```diff
@@ -0,0 +1,43 @@
+//FINDS THE SHORTEST WORD
+function findShortestWord(array) {
+  shortestWord = array.reduce(function (prev, next) {
+    if (prev.length < next.length) {
+      return prev;
+    } else if (prev.length === next.length) {
+      return prev;
+    } else {
+      return next;
+    }
+  });
+  console.log(shortestWord);
+}
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+findShortestWord(danishWords); // returns 'ø'
+
+//FIND AND COUNT THE DANISH LETTERS
+function findAndCountDanishLetters(arr) {
```

Comment:
Maybe a better name for this argument would be "string"? Here the reader thinks it's an array. The solution is fine, just something to think about :)

Reply:
agree)